TaskLayer/MetaMorpheusTask.cs

```diff
@@ -530,22 +530,42 @@ private static string GetExistingFolderWithIndices(IndexingEngine indexEngine, L
                 var directory = new DirectoryInfo(baseDir);
                 DirectoryInfo[] directories = directory.GetDirectories();
 
-                // Look at every subdirectory...
+                // Look at every subdirectory to find indexes folder...
                 foreach (DirectoryInfo possibleFolder in directories)
                 {
-                    if (File.Exists(Path.Combine(possibleFolder.FullName, "indexEngine.params")) &&
-                        File.Exists(Path.Combine(possibleFolder.FullName, "peptideIndex.ind")) &&
-                        File.Exists(Path.Combine(possibleFolder.FullName, "fragmentIndex.ind")) &&
-                        (File.Exists(Path.Combine(possibleFolder.FullName, "precursorIndex.ind")) || !indexEngine.GeneratePrecursorIndex) &&
-                        SameSettings(Path.Combine(possibleFolder.FullName, "indexEngine.params"), indexEngine))
+                    string pathToIndexes = Path.Combine(possibleFolder.FullName, "Indexes");
```

Comment:
We can just create a single folder in the same folder containing the protein database and just call it `[DatabaseName]_Indices_[$DATETIME]`, no need to go looking for it in subdirectories

Reply:
actually, I guess you still have to check for indexEngine.params...